compiler/passes/normalize.cpp

```diff
@@ -2221,21 +2217,23 @@ static void normVarTypeWithInit(DefExpr* defExpr) {
 
     if (isGenericRecordWithInitializers(rhsType)) {
       // Create a temporary with the type specified in the lhs declaration
-      VarSymbol* typeTemp = newTemp("type_tmp");
+      /*VarSymbol* typeTemp = newTemp("type_tmp");
       DefExpr*   typeDefn = new DefExpr(typeTemp);
       CallExpr*  initCall = new CallExpr(PRIM_INIT, typeExpr);
       CallExpr*  initMove = new CallExpr(PRIM_MOVE, typeTemp,  initCall);
 
       defExpr ->insertAfter(typeDefn);
       typeDefn->insertAfter(initMove);
+      */
```

Comment:
I'm going to stop noting commented out code, and call out whatever you miss after a clean up round.  I'll look for other things in the meanwhile

Reply:
(Belatedly, I see that you called this out in the PR message)